Tasks/google-play-release/GooglePlay.ts

```diff
@@ -777,22 +777,36 @@ function resolveGlobPaths(path: string): string[] {
 }
 
 /**
- * Get obb file for the apk from the parent directory.
- * @returns {string[]} path of the obb file if present else null
+ * Get obb file for the apk from the same directory as apk.
+ * @param {string} apkPath apk file path 
+ * @param {string} packageName package name of the apk 
+ * @param {string} versionCode version code of the apk
+ * @returns {string} ObbPathFile of the obb file if present else null
  */
-function getObbFromParentDirectory(apkPath: string): string {
-    const apkDirectory = path.dirname(apkPath);
-    const parentDirectory = path.dirname(apkDirectory);
-    const filenames = fs.readdirSync(parentDirectory);
-    const obbFile: string | undefined = filenames.find(file => path.extname(file) === '.obb');
-
-    if (obbFile !== undefined) {
-        tl.debug(`Found Obb file for upload: ${obbFile}`);
-        return path.join(parentDirectory, obbFile);
+function getObbFromParentDirectory(apkPath: string, packageName: string, versionCode: number): string {
```

Comment:
Could you please change method name according to it's current logic (since it gets from parent or current directory now)?